src/styles/blocks/card.scss

```diff
@@ -0,0 +1,87 @@
+body {
+  margin: 0;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  color: $main-text-color;
+}
```

Comment:
Why does the file created for card block styles have body styles?

Reply:
Thank you for feedback, resolved